### DISCUSSION TITLE
Message has been not received in AWS Lambda or capture by AWS CloudWatch log stream from the AWS                        ActiveMQ Queues

### DISCUSSION THREAD

**Discussion by neetuom** [community/community#134837] [Copilot Conversations]
### Select Topic Area

General

### Body

ActiveMQ message is not received by the AWS Lambda and Cloudwatch logs stream.

     1). Message has been deleted from the ActiveMQ queue but message is not processed by Lambda and neither 
         any exception message is captured by the Cloudwatch log stream.

     2). Message is received from ActiveMQ queue by lambda and processed it successfully but there is no logs 
         in the Cloudwatch log stream.
     
         This issue was faced in production environments.
         
         
         
      We have gone through the ActiveMQ queues from there message has been deleted. There is no sucess or failure of the messages capture by th AWS CloudWatch Logs.

      In another instances message has been successfully processed by the AWS Lambda from the ActiveMQ but there is no trace of the messages in the AWS CloudWatch Log Stream.

      I need your help on the above-mentioned issue that why such issues are crop-up. I have checked ActiveMQ triggers configuration to the Lambda and the IAM Role also. Our application is working fine but there are some instances where we are facing such issues.

**Comment by Akash1134** [comment]
Hi @neetuom ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
